Index: cmake/config-ix.cmake
===================================================================
--- cmake/config-ix.cmake	(revision 326679)
+++ cmake/config-ix.cmake	(working copy)
@@ -153,7 +153,6 @@
         else()
           include_directories(${LIBXML2_INCLUDE_DIR})
         endif()
-        set(LIBXML2_LIBS "xml2")
       endif()
     endif()
   endif()
Index: lib/WindowsManifest/CMakeLists.txt
===================================================================
--- lib/WindowsManifest/CMakeLists.txt	(revision 326679)
+++ lib/WindowsManifest/CMakeLists.txt	(working copy)
@@ -1,7 +1,7 @@
 set(system_libs)
 if( CMAKE_HOST_UNIX )
   if( LLVM_LIBXML2_ENABLED )
-    set(system_libs ${system_libs} ${LIBXML2_LIBS})
+    set(system_libs ${system_libs} ${LIBXML2_LIBRARIES} ${ZLIB_LIBRARIES})
   endif()
 endif()
 
